### PR TITLE
devotionals tabs and scripture!

### LINKED ISSUE #158: Devotionals: Tab View
## Acceptance Criteria

- [ ] It should show devotional / scripture side by side in two tabs
- [ ] User can swipe between tabs

### LINKED ISSUE #159: Devotionals: Scripture
## Acceptance Criteria

- [ ] It should render scripture on the scripture tab in devotionals

This issue should investigate how scripture is currently being rendered in existing app, and implement MVP version of getting scripture rendered in this refactor.

### PR DESCRIPTION
**Pull request by conrad-vanl**
This is meant to be an MVP pass and styles should be cleaned up in #149 

FWIW, the current app isn't as sophisticated in scripture rendering...it completely ignores block quotes and indents, line groups, etc. This at least adds in a first pass add proper indentions and spacing.

Fixes #158 
Fixes #159 

image

image

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

**Comment by richarddubay** [comment]
That iPhone X is looking pretty legit! ❤️

File: src/@ui/Scripture/ScriptureHTMLView.js [hunk 1]
```diff
@@ -16,26 +15,26 @@ const renderer = (node, { children, ...other }) => { // eslint-disable-line
   }
 
   if (className.includes('line-group')) {
-    return <Text>{children}{'\n'}</Text>;
+    return <BodyCopy>{children}{'\n'}</BodyCopy>;
   }
 
   if (className.includes('block-indent')) { // todo
-    return <Text style={{ paddingLeft: 10, paddingTop: 10 }}>{children}</Text>;
+    return <BodyCopy style={{ paddingLeft: 10, paddingTop: 10 }}>{children}</BodyCopy>;
```
Comment (conrad-vanl) [inline src/@ui/Scripture/ScriptureHTMLView.js]: @burkeshartsis did this produce and visual change? These were purposefully using `React-Native` `<Text>` as `<Text>` doesn't pass in any styles, and I was rendering `BodyCopy` at the lowest level possible, here: https://github.com/NewSpring/Apollos/blob/master/src/%40ui/HTMLView/index.js#L40

File: src/@ui/TabView/TabBar/Indicator.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    position.interpolate({
+      inputRange: [0, navigationState.routes.length - 1],
+      outputRange: [0, navigationState.routes.length - 1],
+      extrapolate: 'clamp',
+    }),
+    width,
+  );
+  return (
+    <Animated.View style={{ width, transform: [{ translateX }], ...wrapperStyles }}>
+      <CaretScaled><Caret /></CaretScaled>
```
Comment (burkeshartsis) [inline src/@ui/TabView/TabBar/Indicator.js]: I'm assuming I should see this on Android but I'm not 😢. Tabs switch fine just no animated caret.

Comment (conrad-vanl) [reply src/@ui/TabView/TabBar/Indicator.js]: Has to do with android's overflow issues, I believe

**Pull request merged**